### ISSUE TITLE
[BUG]: Digimon Story Cyber Sleuth - Save Screen Freeze

### ISSUE THREAD

**Issue by thousandvangs** [Vita3K/Vita3K-Android#648]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

v12

### Description of Issue

when saving the game very often it got frozen. Also happen at other instances in the game but when saving is the most often. This is the only problem i found when playing the game. Changing it to low setting like 420p doesn't work too. Saving is very risky in the game since it keeps freeze over and over

### Expected Behavior

No freeze especially when saving file 

### Reproduction Steps

just play the game for a while and try opening save menu in every area. Also happens when a long unskippable cutscenes happen.
downloaded from this site
https://www.indapk.com/2023/02/digimon-story-cyber-sleuth-google-drive.html

### Log File

don't know how to make this

### System Configuration

CPU: Dimensity 8050
GPU/Driver:Mali-G77 MC9
RAM:8gb
OS: Android 13

**Comment by Zekaz21** [comment]
Yes, i get the same problem.. Every time open the save file, freeze.. How can continue to play if saving file always keep freeze everytime

**Comment by diGEDestined** [comment]
I have the same problem too. do you guys know how to fix this? There are times that the game freezes when I save

**Comment by Naichi77** [comment]
Anybody found a fix yet? It seem related to shader compilation when entering the safe menu after leveling or going to new areas and then saving. If you wait in the menu and click through every option before the save option  for 10 seconds and then open every option before clicking save for 20 seconds or so it sometimes works but its not reliable since once you level up maybe twice or going to another area will freeze your game when hitting save. Maybe to much to compile? Anyway, it essentially makes the game unplayable. Its broken.
